src/LuxCore.jl

```diff
@@ -126,6 +126,21 @@ Simply calls `model(x, ps, st)`
 """
 apply(model::AbstractExplicitLayer, x, ps, st) = model(x, ps, st)
 
+"""
+    stateless_apply(model, x, ps, st)
+
+Calls `apply` and only returns the first argument.
+"""
+function stateless_apply(model::AbstractExplicitLayer, x, ps, st)
+    return first(apply(model, x, ps, st))
+end
+
+function stateless_apply(model, x, ps, st)
+    u, st = apply(model, x, ps, st)
+    @assert isempty(st) "Model is not stateless. Use `apply` instead."
+    return u
+end
```

Comment:
If `st` must be empty by definition, should we not define it as 

```julia
function stateless_apply(model::AbstractExplicitLayer, x, ps)
	y, st = apply(model, x, ps, NamedTuple())
	@assert isempty(st) "Model is not stateless. use `apply` instead."
    return y
end
```

Reply:
For the fallback method with `model::Any` would it make sense to use `st=NamedTuple` in the method signature?

Reply:
I don't think there needs to be a `model::Any` dispatch at all.

Reply:
Mostly because we are very clear on https://lux.csail.mit.edu/dev/manual/interface that if it is not subtyped then most things won't be defined.